### ISSUE TITLE
Duplicate id in bookmarks page

### ISSUE THREAD

**Issue by zaymasood** [pulibrary/dpul#1227]
### Error in spec/features/bookmarks_spec.rb
     Bookmarks when checking the bookmark box complies with WCAG
          duplicate-id: id attribute value must be unique (minor)
               https://dequeuniversity.com/rules/axe/4.3/duplicate-id?application=axeAPI
               The following 1 node violate this rule:
           
                    Selector: section
                    HTML: <section id="content" class="col-md-12">
                    Fix any of the following:
                    - Document has multiple static elements with the same id attribute: content

**Comment by thanyabegum** [comment]
Although the bookmarks page duplicate id problem has been fixed via #1263, this is issue is blocked pending https://github.com/projectblacklight/spotlight/issues/2788
